app/scripts/modules/netflix/pagerDuty/pagerDuty.read.service.ts

```diff
@@ -0,0 +1,20 @@
+import {module} from 'angular';
+import {API_SERVICE} from "../../core/api/api.service";
+
+export class PagerDutyReader {
+  static $inject = ['API'];
+
+  constructor(private API: any) {}
+
+  public listServices(): ng.IPromise<any> {
```

Comment:
It makes more sense to move the `PagerDutyService` interface from the component into this service and export it from here (and rename to `IPagerDutyService`, painful as that naming convention is), then modify the return type to `ng.IPromise<[IPagerDutyService]>`. That way, you'll get stronger type inference in the component where you're calling `listServices()`